### ISSUE TITLE
When switching to Google Drive, we flash the 'Upload' button for a second before hiding it

### ISSUE THREAD

**Issue by waferbaby** [cashmusic/platform#639]
You can see it fading out in this screenshot - I've just made the switch from http:// to Google Drive here.

![screen shot 2015-03-19 at 11 59 10 am](https://cloud.githubusercontent.com/assets/1027/6738455/707146ee-ce2f-11e4-8163-76f6028248c3.png)

**Comment by jessevondoom** [comment]
Yeah I'm not sure we can do much better. I'll play around, tho. This is an artifact of a unified interface going from S3 (upload button) to Drive (inserted JS) ...might be able to have it hidden and reveal when S3 loads, which would be more desirable.
